public/utils/constants.js

```diff
@@ -65,7 +65,7 @@ export const DATA_TYPES = {
   KEYWORD: 'keyword',
 };
 
-export const ES_AD_PLUGIN = 'opendistro-anomaly-detection';
+export const ES_AD_PLUGIN = 'opensearch-anomaly-detection';
```

Comment:
minor: change to `OPENSEARCh_AD_PLUGIN`?

Reply:
Good callout. I've seen other `OPENDISTRO-`/`ES-` prefixes in the constants and would require quite a few changes and making sure the references are changed everywhere; will consider this a fix that can be made later and not as part of this PR to try to minimize the changes and de-risk release plans.

Reply:
Opened #10 to track.

Reply:
sure, sounds good